Specs/DateTime/Hindi/TimeExtractor.json

```diff
@@ -0,0 +1,918 @@
+[
+  {
+    "Input": "मैं 7 बजे तक वापस आ जाउंगा",
+        "NotSupportedByDesign": "javascript,python,java",
```

Comment:
Nit: Please fix indentation in json file.

Reply:
fixed